### PR TITLE
Added precheck for required DNS

### PR DESCRIPTION
**Pull request by e-minguez**

### CODE REVIEW

File: pre/dns-hostnames [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      msg "${API} doesn't match the reverse ${IP_API_REVERSE}"
+    fi
+fi
+
+IP_WILDCARD_REVERSE=$(dig +short -x ${IP_WILDCARD})
+if [ -z ${IP_WILDCARD_REVERSE} ]; then
+  errors=$(("${errors}"+1))
+  msg "wildcard reverse not found"
+  else
+    if [ ${IP_WILDCARD_REVERSE} != ${WILDCARD} ]; then
```
Comment (dcritch) [inline pre/dns-hostnames:51]: I don't think you can count on the wildcard IP's reverse to point to foobar. e.g. in one of my IT managed clusters, its wildcard.apps.... while in our lab, the reverse isn't usually resolvable at all.

File: pre/dns-hostnames [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      errors=$(("${errors}"+1))
+      msg "${API} doesn't match the reverse ${IP_API_REVERSE}"
```
Comment (dcritch) [inline pre/dns-hostnames:53]: should be wildcard doesn't match, not API

**Review by dcritch — CHANGES_REQUESTED** [review]

File: pre/dns-hostnames (hunk 1 above)
Comment (e-minguez) [reply pre/dns-hostnames]: You are right, I commented out the wildcard reverse.

File: pre/dns-hostnames (hunk 2 above)
Comment (e-minguez) [reply pre/dns-hostnames]: You are right, I fixed it but it is already commented out.

**Comment by e-minguez** [comment]
I've made the errors red and the 'warnings' yellow for this check as well.

**Comment by dcritch** [comment]
thanks!

**Pull request merged**